extensions/amp-next-page/0.1/test/test-amp-next-page.js

```diff
@@ -124,4 +139,60 @@ env => {
     win.dispatchEvent(new Event('scroll'));
     yield macroTask();
   });
+
+  it('adds the hidden class to hideSelector elements', function* () {
+    const exampleDoc = createExampleDocument(doc);
+    xhrMock.expects('fetchDocument')
+        .returns(Promise.resolve(exampleDoc))
+        .once();
+
+    const nextPageService = getService(win, 'next-page');
+    const attachShadowDocSpy =
+        sandbox.spy(nextPageService.multidocManager_, 'attachShadowDoc');
+
+    sandbox.stub(viewport, 'getClientRectAsync').callsFake(() => {
+      // 1x viewport away
+      return Promise.resolve(
+          layoutRectLtwh(0, 0, sizes.width, sizes.height * 2));
+    });
+
+    win.dispatchEvent(new Event('scroll'));
+    yield macroTask();
+
+    const shadowDoc = attachShadowDocSpy.firstCall.returnValue.ampdoc;
```

Comment:
@aghassemi @peterjosling This test is causing widespread unit test failures on `master` today: https://travis-ci.org/ampproject/amphtml/jobs/377848593#L2008

Can you fix this test, or skip if the fix isn't obvious?